### PR TITLE
 Task: Create folder for project in Repository

### LINKED ISSUE #4: Create folder for project in repository.
create following project structure:

- make folder **TechCrunch/projects**
- create README.md file there
- make folder **TechCrunch/projects/news_feed**
- create README.md file there

### PR DESCRIPTION
**Pull request by Raspberry-beans**
-Created project folder
 	-Created README.md file for project folder
 	-Created new project folder namde "news_feed"
 	-Created README.md file for new_feed project

 Relates: 4

### CODE REVIEW

**Review by M-Shan — CHANGES_REQUESTED** [review]
thanks for your first commit on the project 👍 
following are my comments:

- in **README.md** files write the folder purpose e.g. in project **README.md** file write that this folder will contain different projects code scripts, resources, artifacts etc.

in order to understand how to write markdown files (.md) read [this](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax).

in order to see how your markdown file will look like you can use [this ](https://markdownlivepreview.com/) website.

**Comment by M-Shan** [comment]
in order to change your GitHub theme to dark mode then search for **GitHub Dark Theme** at [Chrome Web Store](https://chrome.google.com/webstore/category/extensions).

**Comment by Raspberry-beans** [comment]
> thanks for your first commit on the project 👍
> following are my comments:
> 
> * in **README.md** files write the folder purpose e.g. in project **README.md** file write that this folder will contain different projects code scripts, resources, artifacts etc.
> 
> in order to understand how to write markdown files (.md) read [this](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax).
> 
> in order to see how your markdown file will look like you can use [this ](https://markdownlivepreview.com/) website.

**Comment by Raspberry-beans** [comment]
@M-Shan I have addressed the required changes in markdown files. Some of the syntax does not appear as it should be. Kindly help me to clear this issue!

**Comment by M-Shan** [comment]
> @M-Shan I have addressed the required changes in markdown files. Some of the syntax does not appear as it should be. Kindly help me to clear this issue!

ok.
i see what's wrong --> you need to add a space after `#` for headings.
and for links there shall be no space after `]` --> an example is as follow: 
`[M-Shan TechCrunch projects](https://github.com/M-Shan/TechCrunch/projects)`

File: projects/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Projects
+## This file contains useful information regarding current projects
+
+
+##Contents available in projects folder
```
Comment (M-Shan) [inline projects/README.md]: add space after `##`

File: projects/news_feed/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Sports News Feed
```
Comment (M-Shan) [inline projects/news_feed/README.md]: add space after `#`

File: projects/news_feed/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+##The purpose of this project is to inform its users about everydays sports events and news using a specially designed GUI (Graphical User Interface) system.
```
Comment (M-Shan) [inline projects/news_feed/README.md]: add space after `##`

File: projects/news_feed/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+###Further advancement and code scritps will be shared soon
```
Comment (M-Shan) [inline projects/news_feed/README.md]: add space after `###`

File: projects/news_feed/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+You can check the status of project by clicking on [M-Shan TechCrunch projects] (https://github.com/M-Shan/TechCrunch/projects)
```
Comment (M-Shan) [inline projects/news_feed/README.md]: remove space --> `unch projects](https`

**Review by M-Shan — CHANGES_REQUESTED** [review]

**Comment by Raspberry-beans** [comment]
@M-Shan All the issues related to README files have been resolved.
However, today I have noticed that my commits are mentioned with my old github ID faheem-rehman not Raspberry-beans
Can you find out where the actual problem is?
Thanks!

**Comment by M-Shan** [comment]
> oticed that

im not sure how to address this issue.
if your account is not used then delete that one.

and after commit you can see wht's the name in commit message via `git log `

**Review by M-Shan — APPROVED** [review]
good work 👍

**Pull request merged**